CSS/StyleSheet.css

```diff
@@ -77,6 +77,35 @@ header .overlay {
   z-index: 1;
 }
 
+.dnd-content {
+  color: white;
+  padding-left: 25px;
+}
+
+.dnd-bg {
+  background-color: black;
+}
+
+.dnd-dropdown {
+  border: 5px solid;
+  background-color: black;
+  font-size: 25px;
+}
+
+.dnd-menu button {
+  color: white;
+}
+
+.dnd-card {
+  background-color: black;
+  color: white;
+}
+
+.custom-container {
+  padding-left: 25px;
+  padding-right: 50px;
```

Comment:
while pixel-paddings are fine for the Devsoc website, be aware that typically it's better to use `rem` (so that the padding is relative to font sizing)